Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -274,6 +276,35 @@ public HttpResponseMessage MiniDump(int id, int dumpType = 0, string format = nu
             }
         }
 
+        [HttpPost]
+        public HttpResponseMessage TakeCrashDump(int id, bool includeMemory = false)
+        {
+            using (_tracer.Step("ProcessController.TakeCrashDump"))
+            {
+                try
+                {
+                    var process = GetProcessById(id);
+                    var exe = new Executable(ResolveProcDumpPath(), _environment.CrashDumpsPath, TimeSpan.MaxValue);
+                    var memoryFlag = includeMemory ? "-ma" : string.Empty;
+
+                    Task.Run(() => exe.ExecuteReturnExitCode(_tracer, s => _tracer.Trace(s), s => _tracer.TraceError(s), $"-accepteula -t {process.Id} -e -g {memoryFlag}"));
```

Comment:
No, you need to pass -64 if you want to capture a 64 bit dump for a 32 bit process. I guess if we pass that on 32 bit process, that'll make all the dumps 64 bit and we can always use the x64 debugger. I'm not sure if we lose anything from that.